src/main/java/org/entando/kubernetes/client/model/assembler/InstallPlanAssembler.java

```diff
@@ -1,5 +1,5 @@
-/**
- * assembler pattern to adapt AnalysisReport to InstallPlan.
+/*
```

Comment:
@firegloves revert this pls

avoid committing a file if there is not functional change